sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/MessageUtilsTest.java

```diff
@@ -60,4 +68,33 @@ void convertUUIDToDotNetBytes() {
         // Assert
         assertArrayEquals(dotNetGuidBytes, convertedBytes, "UUID conversion from Java to DotNet failed");
     }
+
+    @ParameterizedTest
+    @MethodSource("calcTotalTimeoutTestData")
+    void calcTotalTimeout(List<Object> testData) {
+        AmqpRetryOptions amqpRetryOptions = (AmqpRetryOptions) testData.get(0);
+        long expectedResult = (long) testData.get(1);
+        assertEquals(MessageUtils.calcTotalTimeout(amqpRetryOptions).toMillis(), expectedResult);
```

Comment:
The order is `assertEquals(expected, actual)`

Reply:
Updated